source/Xunit.Gherkin.Quick/CoreModel/StepMethod/ScenarioStepPattern.cs

```diff
@@ -7,18 +7,39 @@ namespace Xunit.Gherkin.Quick
     internal sealed class ScenarioStepPattern
     {
         private readonly string _pattern;
-        public string Pattern { get { return _pattern; } }
 
+        /// <summary>
+        /// Original pattern as supplied from step attributes.
+        /// </summary>
+        public string Pattern { get { return _pattern; } }

```

Comment:
If you added RegexPattern, please rename this to OriginalPattern for clarity. Comment helps, but the name can be also improved.

Reply:
Done